Majasdarbs/03-majasdarbs/portfolio/css/style.css

```diff
@@ -0,0 +1,47 @@
+/* HEADER START*/ 
+#header  nav  ul {
+    display: flex;
+    align-items: center;
+    position: fixed;
+    justify-content: space-between;
+    background-color: darkgrey;
+    left: 0px;
+    right: 0;
+    height: 40px;
+    width: 100%;
+}
+ 
+#header nav ul li:hover {
+    background-color: darkmagenta;
+    font-weight: bold;
+}
+     
+
+
+#header a {
+    color: whitesmoke;
+    list-style: none;
+}
+/* HEADER END */
+
+/* MAIN */
+main {
+    padding: 40px;
+    background-image: url("https://i.graphicmama.com/blog/wp-content/uploads/2020/01/24160451/free-carbon-fiber-pattern-backgrounds.jpg");
```

Comment:
check later if you need to add
background-size: cover;
background-repeat: no-repeat;
background-position: center;